### PR TITLE
[TE] report-missing-anomaly from dashboard base functionality

### PR DESCRIPTION
**Pull request by newsummit**
In this pass:
- adding "source" attribute to anomaly model (in order to display user-reported anomalies in dashboard)
- putting in a fix for /manage/alerts/performance (bug was omitting the full set of applications in the report table)
- adding comment field to display of user-reported anomalies in both dashboard and alert page

Dashboard view:
screen shot 2018-08-23 at 6 27 52 pm

Alert page view:
screen shot 2018-08-23 at 6 32 00 pm

>> all tests passing !!!

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/dimensions/template.hbs [hunk 1]
```diff
@@ -3,6 +3,15 @@
...
       {{record.anomaly.functionName}}
     {{/link-to}}
+
+    {{!--  TODO: move this into a link above each metric table - Report missing anomaly link
+    {{#link-to "manage.alert.explore" record.anomaly.functionId (query-params duration=record.queryDuration startDate=record.queryStart endDate=record.queryEnd openReport=true) class="te-anomaly-table__link te-anomaly-table__link--glyph" target="_blank"}}
+      <i class="glyphicon glyphicon-edit"></i>
+      {{#tooltip-on-element class="te-anomaly-table__tooltip"}}
+        Report missing anomaly for this alert.
+      {{/tooltip-on-element}}
+    {{/link-to}} --}}
```
Comment (apucher) [inline thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/dimensions/template.hbs]: should this still be here?

File: thirdeye/thirdeye-frontend/app/pods/manage/alerts/performance/route.js [hunk 2]
```diff
@@ -217,6 +217,10 @@ export default Route.extend({
     const groupsWithAppName = activeGroups.filter(group => isPresent(group.application));
     const groupsWithAlertId = groupsWithAppName.filter(group => group.emailConfig.functionIds.length > 0);
     const filteredGroups = isDemoMode ? groupsWithAlertId.slice(0, 3) : groupsWithAlertId;
+
+    // NOTE: use this in order to find non-existent alert in the event of an error
+    // filteredGroups.filter(group => group.emailConfig.functionIds.includes(45639479)));
```
Comment (apucher) [inline thirdeye/thirdeye-frontend/app/pods/manage/alerts/performance/route.js]: still needed?

File: thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/dimensions/template.hbs (hunk 1 above)
Comment (newsummit) [reply thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/dimensions/template.hbs]: I just want it there for the next iteration - didn't want to sink time into it this time.

File: thirdeye/thirdeye-frontend/app/pods/manage/alerts/performance/route.js (hunk 2 above)
Comment (newsummit) [reply thirdeye/thirdeye-frontend/app/pods/manage/alerts/performance/route.js]: Yes

**Pull request merged**